File: Java/1010.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        int prod1 = leitor.nextInt();
        int qntd1 = leitor.nextInt();
        double preco1 = leitor.nextDouble();
        int prod2 = leitor.nextInt();
        int qntd2 = leitor.nextInt();
        double preco2 = leitor.nextDouble();
        double tot = (qntd1 * preco1) + (qntd2 * preco2);
        
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", tot);
    }
 
}

File: Python/1010.py
# -*- coding: utf-8 -*-
prod1, qntd1, preco1 = input().split(" ")
prod2, qntd2, preco2 = input().split(" ")

prod1 = int(prod1)
qntd1 = int(qntd1)
preco1 = float(preco1)

prod2 = int(prod2)
qntd2 = int(qntd2)
preco2 = float(preco2)

tot = (qntd1 * preco1) + (qntd2 * preco2)
        
print("VALOR A PAGAR: R$ %.2f" % tot)